### PR TITLE
Zeno pcb V0.0.4 update via + fileformatting

### PR DESCRIPTION
**Pull request by paulgali**
## Description

- Added new via compatibility
- Updated layout of info.json and removed deprecated lines from config.h, etc
- Updated bottom row layout options
- Updated reset instructions in readme

## Types of Changes

- [x] Core
- [x] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* QMK lint was not passing, now working as intended

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/skme/zeno/info.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
...":"2,2", "x":-0.25, "y":3.3000000000000003}, {"label":"2,3", "x":0.75, "y":3.3000000000000003}, {"label":"2,4", "x":1.75, "y":3.3000000000000003}, {"label":"2,5", "x":2.75, "y":3.3000000000000003}, {"label":"3,2", "x":0.25, "y":4.3}, {"label":"3,3", "x":1.25, "y":4.3}, {"label":"3,4", "x":2.25, "y":4.3}, {"label":"3,5", "x":3.25, "y":4.3}, {"label":"4,3", "x":0.5, "y":5.3}, {"label":"4,5", "x":-7.8, "y":8.45, "w":2.75}, {"label":"0,7", "x":-3.75, "y":3.3}, {"label":"0,8", "x":-2.75, "y":3.3}, {"label":"0,9", "x":-1.75, "y":3.3}, {"label":"0,10", "x":-0.75, "y":3.3}, {"label":"1,6", "x":-4.25, "y":4.3}, {"label":"1,7", "x":-3.25, "y":4.3}, {"label":"1,8", "x":-2.25, "y":4.3}, {"label":"1,9", "x":-1.25, "y":4.3}, {"label":"2,6", "x":-3.95, "y":5.3}, {"label":"2,7", "x":-2.95, "y":5.3}, {"label":"2,8", "x":-1.9500000000000002, "y":5.3}, {"label":"2,9", "x":-0.9500000000000002, "y":5.3}, {"label":"3,6", "x":-4.45, "y":6.3}, {"label":"3,7", "x":-3.45, "y":6.3}, {"label":"3,8", "x":-2.45, "y":6.3}, {"label":"3,9", "x":-1.4500000000000002, "y":6.3}, {"label":"4,7", "x":-4.45, "y":7.3, "w":2}, {"label":"4,8", "x":-2.45, "y":7.3}, {"label":"4,9", "x":-1.4500000000000002, "y":7.3, "w":1.25}]
```
Comment (zvecr) [inline keyboards/skme/zeno/info.json:16]: A precision of `0.30000000000000004` here is rather pointless....

File: keyboards/skme/zeno/config.h [hunk 2]
```diff
@@ -17,13 +17,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
+/* key matrix size */
```
Comment (zvecr) [inline keyboards/skme/zeno/config.h:20]: You shouldnt be putting this and many others back, they have been removed on purpose.

**Review by zvecr — CHANGES_REQUESTED** [review]
Submodule changes also need reverting

File: keyboards/skme/zeno/info.json (hunk 1 above)
Comment (paulgali) [reply keyboards/skme/zeno/info.json]: Precision updated to 4 trailing digits in following commit

File: keyboards/skme/zeno/config.h (hunk 2 above)
Comment (paulgali) [reply keyboards/skme/zeno/config.h:18]: Pragma removed. I tried moving matrix configs to info but i could not compile firmware once i had done it.

Comment (zvecr) [reply keyboards/skme/zeno/config.h:18]: No, you are adding back matix size, debounce, polling interval, mcu, bootloader. All of the above are wrong additions and need to be reverted.

You should really review your proposed changes under the "Files Changed" tab.

Comment (paulgali) [reply keyboards/skme/zeno/config.h]: This is fixed

**Comment by paulgali** [comment]
Cannot compile without zeno.c and zeno.h even after moving everything into info.json. Is this expected?

Also cannot get anything to show in via despite a valid file working on desktop, if im missing something can you advise?

**Comment by zvecr** [comment]
> Submodule changes also need reverting

**Comment by paulgali** [comment]
> 

Done, my mistake here.

QMK link passes locally. Via confirmed to work as intended. All necessary parts are moved to info.json. Apologies for the mess, hope everything is good on this now.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**